inc/class-core-sitemaps-renderer.php

```diff
@@ -109,7 +113,14 @@ public function render_index( $sitemaps ) {
 	 * @return string|false A well-formed XML string for a sitemap index. False on error.
 	 */
 	public function get_sitemap_index_xml( $sitemaps ) {
-		$sitemap_index = new SimpleXMLElement( '<?xml version="1.0" encoding="UTF-8" ?>' . $this->stylesheet_index . '<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"></sitemapindex>' );
+		$sitemap_index = new SimpleXMLElement(
```

Comment:
Note: this change (and the equivalent below) is not necessary to achieve this goal, it's just "coding standards" related...I find it easier to see what's going on using `sprintf()` than string concatenation to construct the argument to `new SimpleXMLElement()`.